configCivcraft.yml

```diff
@@ -695,9 +697,9 @@ recipes:
   Craft_Anvils_Advanced:
     type: PRODUCTION
     name: Craft Anvils
-    production_time: 5s
+    production_time: 10s
     input:
-      iron blocks:
+      iron block:
```

Comment:
Hmm last time I tried using spaces in yaml identifers it crashed, but I just ran this config through a validator and it seems fine, not sure. The yaml identifier is no longer the identifier used ingame, we now have something centralized for that. You can use whatever you want for the yaml structure, as long as the identifiers are unique.